src/main/java/org/openstreetmap/atlas/checks/database/DatabaseConnection.java

```diff
@@ -29,6 +34,7 @@
     public DatabaseConnection(final String connectionUrl)
```

Comment:
We could annotate ```connectionUrl``` with the @javax.validation.constraints.NotNull tag since it looks like we'd get unexpected behavior if ```connectionUrl``` is null. DatabaseConnection is a public class so we would want to avoid any misuse when creating objects

```public DatabaseConnection(@NotNull final String connectionUrl)```

Reply:
Passing in a null value for the connection url will throw a jdbc SQLException
```java
12:03:36.184 [main] DEBUG o.o.a.u.c.a.AbstractAtlasShellToolsCommand - Command using context 3
Oct 30, 2019 12:03:36 PM org.postgresql.Driver parseURL
WARNING: JDBC URL must contain a / at the end of the host or port: jdbc:postgresql://null
Oct 30, 2019 12:03:36 PM org.postgresql.Driver parseURL
WARNING: JDBC URL must contain a / at the end of the host or port: jdbc:postgresql://null
12:03:36.213 [main] ERROR o.o.a.c.d.FlagDatabaseSubCommand - Invalid connection string. host[:port]/database
java.sql.SQLException: No suitable driver found for jdbc:postgresql://null
	at java.sql.DriverManager.getConnection(DriverManager.java:689) ~[na:1.8.0_161]
	at java.sql.DriverManager.getConnection(DriverManager.java:270) ~[na:1.8.0_161]
```